addons/website_slides/models/slide_channel.py

```diff
@@ -533,21 +566,12 @@ def _get_categorized_slides(self, base_domain, order, force_void=True, limit=Fal
                 'total_slides': len(category_slides),
                 'slides': category_slides[(offset or 0):(limit + offset or len(category_slides))],
             })
-        # Then add uncategorized slides
-        uncategorized_slides = all_slides.filtered(lambda slide: not slide.category_id)
-        if uncategorized_slides or force_void:
-            category_data.append({
```

Comment:
You could just `insert(0, {...})` instead of append to minimize diff in stable, so you don't have to move the whole block.